lib/mix/tasks/ecto.rollback.ex

```diff
@@ -40,6 +40,8 @@ defmodule Mix.Tasks.Ecto.Rollback do
 
   """
 
+  @recursive true
```

Comment:
This can be also dangerous? We want to rollback one and then we end-up rolling back all of them.